examples/complete-vpc/outputs.tf

```diff
@@ -1,60 +1,60 @@
 # VPC
 output "vpc_id" {
   description = "The ID of the VPC"
-  value       = "${module.vpc.vpc_id}"
+  value       = module.vpc.vpc_id
 }
 
 # Subnets
 output "private_subnets" {
   description = "List of IDs of private subnets"
-  value       = ["${module.vpc.private_subnets}"]
+  value       = [module.vpc.private_subnets]
```

Comment:
Not sure if this really need to be a array/list. I think passing the variable directly should be fine but not sure.

Reply:
Indeed. Corrected the rest of the examples as well.